web/angular-wallet/src/app/main/messages/messages.component.html

```diff
@@ -11,7 +11,7 @@
         </mat-sidenav>
         <message-start *ngIf="!selectedMessage"></message-start>
         <message-view *ngIf="selectedMessage"
-                      [feeNQT]="rightSidenav && rightSidenav.options && rightSidenav.options.feeNQT"
+                      [feeNQT]="rightSidenav && rightSidenav.options && rightSidenav.options.fee"
```

Comment:
Already reverting this.... fuck renaming in Webstorm, often screws up everything